### PR TITLE
Improve formal introduction

### PR DESCRIPTION
**Pull request by szepeviktor**
@adamlaki All the changes presume formal tone.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 🌲 **Our Commitment:** We're a passionate two-person web development studio dedicated to producing sustainable, future-proof code for the web. Whether your project is small or large, we ensure it's built to last. Our core values and deliverables are: simple and clean code, reliability, clear communication, two-way partnership.
 
-**</> WordPress:** We code truly custom WP themes, develop custom plugins and WooCommerce integrations.
```
Comment (szepeviktor) [inline profile/README.md:17]: "" or "WordPress"

File: profile/README.md [hunk 2]
```diff
@@ -14,11 +14,11 @@
...
+**</> WordPress:** We code truly custom themes, develop custom plugins and WooCommerce integrations.
 
-🌱 **Open-source:** We create and manage multiple open-source projects. Check them out: [Root](https://github.com/conedevelopment/root), [Bazar](https://github.com/conedevelopment/bazar), and [Spruce CSS](https://github.com/conedevelopment/sprucecss).
+🌱 **Open Source:** We create and maintain several open-source projects. Check them out: [Root](https://github.com/conedevelopment/root), [Bazar](https://github.com/conedevelopment/bazar), and [Spruce CSS](https://github.com/conedevelopment/sprucecss).
```
Comment (szepeviktor) [inline profile/README.md:19]: github.com main menu

image

Comment (szepeviktor) [reply profile/README.md:19]: repo -> maintain

File: profile/README.md [hunk 3]
```diff
@@ -14,11 +14,11 @@
...
 
-📫 **Get in Touch:** Need help with a project, want to hire a reliable, long-term subcontractor, or just say hello? Email us at [hello@conedevelopment.com](mailto:hello@conedevelopment.com).com. We'd love to hear from you!
+📫 **Get in touch:** Need help with a project, want to hire a reliable, long-term subcontractor, or just want to say hello? Send us an email at [hello@conedevelopment.com](mailto:hello@conedevelopment.com).com. We'd love to hear from you!
```
Comment (szepeviktor) [inline profile/README.md:21]: "want to" -> add verb

Comment (szepeviktor) [reply profile/README.md:21]: "Send us" -> more formal

**Comment by adamlaki** [comment]
Viktor, why do you think that the informal tone is bad?

**Comment by adamlaki** [comment]
Thank you, Viktor!

**Comment by szepeviktor** [comment]
> Viktor, why do you think that the informal tone is bad?

I have no such thought!

**Pull request merged**